playbooks/roles/homebrew/tasks/main.yml

```diff
@@ -3,7 +3,9 @@
   register: homebrew_installed
 
 - name: Install Homebrew
-  shell: ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  shell: |
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    /usr/local/bin/brew bundle
```

Comment:
This command will fail with "`No Brewfile found.`. `--file=- </dev/null` makes it a proper no-op.

Reply:
Could do `brew tap homebrew/bundle` instead.

Reply:
:heavy_check_mark: I've replaced `/usr/local/bin/brew bundle` with `/usr/local/bin/brew tap homebrew/bundle`. Thanks @native-api and @MikeMcQuaid!